addon/xml/templates/street.js

```diff
@@ -1,13 +1,15 @@
 // prettier-ignore
 export const getStreet =
-`{{#>base}}
+  `{{#>base}}
   <ns2:getStreet>
     {{#if model.description.descriptionLong}}
       <ns2:description>
         <ns2:language>{{model.language}}</ns2:language>
         <ns2:descriptionLong>{{model.description.descriptionLong}}</ns2:descriptionLong>
       </ns2:description>
     {{/if}}
-    {{> Locality model=model.locality}}
+    {{#if model.locality.nameLong}}
+      {{> Locality model=model.locality}}
+    {{/if}}
```

Comment:
Locality name is optional, therefore we won't include an empty `nameLong` if not given, since it would produce validation errors on the API.